TRANSLATION.md

```diff
@@ -0,0 +1,77 @@
+# reactjs.org saytining O'zbek tilidagi tarjimasi
+
+Hozirgi jarayon: 
+https://github.com/reactjs/uz.reactjs.org/issues/1
+
+## Tarjima bo'yicha kelishuv
+
+| Terminning asli | Tarjim |
+| ------------------ | ---------- |
+| API reference | API ma'lumotnoma |
+| array | massiv |
+| arrow function |  arrow funksiya, strelka funksiya  |
+| attribute | atribut |
+| batch | yangilash guruhi |
+| batching | guruhlash |
+| bug | bag, xatolik  |
+| bundler | bandler |
+| callback | kolbek |
+| camelCase | *camelCase* |
+| class component | sinf komponentasi, class komponent |
+| (un)controlled component | boshqaril(may)adigan komponent |
+| debugging | debag qilish |
+| DOM container | DOM-konteyner |
+| error | xatolik |
+| feature | imkoniyat |
+| development | dasturlash |
+| development mode | dasturlash rejimi |
+| framework | freymvork |
+| function component | funksiyali komponent |
+| hook | ho'k, ilgak |
+| incapsulation | inkapsulyatsiya |
+| incapsulated | inkapsulyatsiya qilingan |
+| key | kalit |
+| lazy initialization | lezy boshlanish, dangasa boshlanish |
+| library | kutubxona |
+| lifecycle | hayotiy sikl |
+| lifecycle method | hayotiy jarayon metodi |
+| lifting state up | ko'tarish holati |
+| local state | ichki holat, lokal holat |
+| lowercase | kichik harf, *lowercase* |
+| mock | mok, hayoliy |
+| (im)mutable | (o'zgarmas)/o'zgaruvchan |
+| (im)mutability | (o'zgarmaslik)/o'zgaruvchanlik |
+| (un)mount | mount (qilmaslik)/qilish |
+| mutate | mutatsiya qilmoq, o'zgartirmoq |
+| Note | Eslatma |
+| online | onlayn |
+| online playground | onlayn maydon |
+| paint | chizmoq, chizish, bo'yash |
+| prop | prop *(birlik)* |
+| props | props *(ko'plik)* |
+| production | prodakshin |
+| production mode | prodakshin rejim |
+| reuse | qayta ishlatish, qayta foydalanish |
+| React | React |
+| React element | React element |
+| reconciliation | kelishuv, muvofiqlashtirish |
+| render(ing) | chizmoq (chizish) |
```

Comment:
What about `ko'rsatish` ?

Reply:
@rustamwin you can add your own variant.